ferrocene/doc/internal-procedures/src/working-with-the-ci.rst

```diff
@@ -112,6 +114,15 @@ For example, to run our licensing checks you can run:
    
    uv run reuse --include-submodules lint
 
+Or, to run the ``split-tasks.py`` script for debugging:
+
+.. code-block:: bash
+   
+   uv run ferrocene/ci/split-tasks.py test
+
+In CI workflows you should always use ``uv run`` for calling Python. In the case of ``make.py`` it
```

Comment:
```suggestion
In CI workflows you should always use ``uv run`` for calling Python. In the case of ``x.py`` it
```

Reply:
Doesn't `make` do it as well, by calling its own file of requirements?